libs/geography/src/reducers/country-entities/country-entities.reducer.ts

```diff
@@ -6,31 +6,35 @@ import { DaffCountry } from '../../models/country';
 /**
  * Interface for country entity state.
  */
-export interface DaffCountryEntityState extends EntityState<DaffCountry> {}
+export interface DaffCountryEntityState<T extends DaffCountry> extends EntityState<T> {}
 
 /**
  * Country Adapter for changing/overwriting entity state.
  */
-export const countryAdapter: EntityAdapter<DaffCountry> = createEntityAdapter<DaffCountry>();
+export const getCountryAdapter = (() => {
```

Comment:
I think this should go in a different file, since it's used to set state and to access selectors

Reply:
I moved a lot of the reducer stuff into separate files